app/Http/Controllers/Api/V1/GroupController.php

```diff
@@ -80,6 +80,33 @@ public function index(Organization $suborganization)
         ], 200);
     }
 
+    /**
+     * Get All Organization Groups
+     *
+     * @urlParam suborganization required The Id of a suborganization Example: 1
+     * Get a list of the groups of an organization.
+     *
+     * @responseFile responses/group/group.json
+     *
+     * @return Response
+     */
+    public function getOrgGroups(Organization $suborganization)
+    {
+        $this->authorize('viewAny', [Group::class, $suborganization]);
+
+        $groups = $this->groupRepository->getOrgGroups($suborganization->id);
```

Comment:
I think we should combine getOrgGroups and getGroupDetail and take this code in repository.